docs/cli.md

```diff
@@ -0,0 +1,13 @@
+You can manipulate KubeSphere DevOps via CLI.
+
+## Create Pipeline
+
+```shell
+ks pip create --ws simple --template java --project default --skip-check -b good
```

Comment:
Before we use ks, I suggest introducing `ks` from `docs/installation.md#kubesphere-cli`.